tests/disas/winch/x64/br_if/save_state_before_br_emission.wat

```diff
@@ -86,15 +86,13 @@
 ;;       movss   0x23(%rip), %xmm15
 ;;       movss   %xmm15, (%rsp)
 ;;       movq    4(%rsp), %rax
-;;       movl    (%rsp), %r11d
+;;       movss   (%rsp), %xmm15
 ;;       addq    $4, %rsp
-;;       movl    %r11d, (%rax)
+;;       movss   %xmm15, (%rax)
```

Comment:
This is a result of changing how memory values are popped from the stack. It now uses the type of the Wasm value to determine the scratch register to use so this now uses the XMM scratch register instead of a GPR.